tests/unit_tests/dft/include/compute_inplace.hpp

```diff
@@ -94,9 +94,12 @@ int DFT_Test<precision, domain>::test_in_place_buffer() {
             auto acc_host = inout_buf.get_host_access();
             auto ptr_host = reinterpret_cast<FwdOutputType*>(acc_host.get_pointer());
             for (std::int64_t i = 0; i < batches; i++) {
-                EXPECT_TRUE(check_equal_strided<domain == oneapi::mkl::dft::domain::REAL>(
```

Comment:
Looks good to me, this is the only point that I prefer the old syntax as it's slightly more readable, but as these are unit test its fine. 
Thank you for this effort.